### ISSUE TITLE
shopify populate product command is not working

### ISSUE THREAD

**Issue by asoni11999** [Shopify/shopify-cli#1436]
# Issue summary

Write a short description of the issue here ↓

I am trying to add product in my app but it is giving me this. I try to resolve this whole day but I did not get any solution for this problem.

Please let me me know if anyone have answer for this.

## Actual behavior
X Command not allowed with current login. Please check your login details with shopify whoami. You may need to request additional permissions for this action.

**Comment by gomarion** [comment]
I experienced this issue a while ago, and I noticed that this happens if I login via the account I used to create the development store. So I created a new account and added all permissions for that new account. Then in shopify-cli, I login to the store using that new account.

**Comment by asoni11999** [comment]
Did not get what do you mean? Do I have to sign as a staff member or something else.

**Comment by gomarion** [comment]
Yes, that's what I did. Aside from my main partners account I used to create the development store, I created a new account which I assigned as staff with all the permissions available, then in my cli, I logged in (shopify login) using that second account. Then after that I was able to successfully populate products.

**Comment by asoni11999** [comment]
I logged in using staff account in cli but it is showing:

It doesn't appear that you're logged in. You must log into a partner organization or a store staff account.

Do I have to create a new partner account with staff email id?? please reply.

**Comment by gomarion** [comment]
@asoni11999 In my case, it wasn't even a partner account. I just added a staff account with a new email address and logged in CLI with that new staff account. If you're in a Windows machine, try installing using the Windows version. I've had more success using that one instead of WSL at this moment.

**Comment by bethkellyshopify** [comment]
I had the same problem with the same workaround.  This is pretty painful if you are a solo app developer.  It seems like the app doesn't recognize "owner" as someone with all the privileges.
